addons/web/static/src/views/fields/properties/property_value.js

```diff
@@ -0,0 +1,274 @@
+/** @odoo-module **/
+
+// Represent one property value
+// Supports many types and instantiates the appropriate component for it
```

Comment:
Again, should probably not be here but as Class docstring instead?

(And would be nice to list all supported types).